kfdef/kfctl_openshift.yaml

```diff
@@ -0,0 +1,27 @@
+
+apiVersion: kfdef.apps.kubeflow.org/v1beta1
```

Comment:
@crobby there is an extra line on top, can you please delete it?

Reply:
Done